pkg/api/controllers/swagger.yaml

```diff
@@ -822,6 +822,125 @@ paths:
                   error_message:
                     type: string
                     example: "conflict error on reference"
+  /{ledger}/balances:
+    get:
+      tags:
+        - balances
+      summary: Get the balances from a ledger's account
+      operationId: getBalances
+      parameters:
+        - name: ledger
+          in: path
+          description: Name of the ledger.
+          required: true
+          schema:
+            type: string
+            example: ledger001
+        - name: address
+          in: query
+          description: Filter balances involving given account, either
+            as source or destination.
+          schema:
+            type: string
+            example: users:001
+        - name: after
+          in: query
+          description: Pagination cursor, will return accounts after given address, in descending order.
+          schema:
+            type: string
+            example: users:003
+        - name: pagination_token
+          in: query
+          description: |-
+            Parameter used in pagination requests. 
+            Set to the value of next for the next page of results. 
+            Set to the value of previous for the previous page of results.
+          schema:
+            type: string
+            example: aHR0cHM6Ly9nLnBhZ2UvTmVrby1SYW1lbj9zaGFyZQ==
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                required:
+                  - cursor
+                properties:
+                  cursor:
```

Comment:
This response has one extra level

Reply:
@antoinegelloz Maybe i'm tired, but the response seem ok for me. Can you point the error.

Reply:
Here is the good format:

<img width="404" alt="Screenshot 2022-07-06 at 18 45 01" src="https://user-images.githubusercontent.com/42968436/177601784-530d2722-41b3-49eb-9e1c-674124b3f15d.png">

Reply:
@antoinegelloz ok I thought you said there was a level missing when in fact there is one too many.